docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -26,35 +26,41 @@ weight: 80
 
 # Configure data links
 
-Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization shows four servers, you can add a data link to one or two of them. You can also link panels using data links.
```

Comment:
Do we generally prefer present tense in docs?

Reply:
Yes!